src/NuGet.Services.EndToEnd/Support/Clients/Clients.cs

```diff
@@ -94,8 +93,34 @@ private static Clients InitializeInternal(TestSettings testSettings)
                 symbolServerClient);
         }
 
-        private static IRetryingAzureManagementAPIWrapper GetAzureManagementAPIWrapper(TestSettings testSettings)
+        private static IRetryingAzureManagementAPIWrapper GetAzureManagementAPIWrapperForGallery(TestSettings testSettings)
         {
+            if (testSettings.GalleryConfiguration.AzureManagementAPIWrapperConfiguration != null)
```

Comment:
these methods could be simplified if you have one helper that took a `AzureManagementAPIWrapperConfiguration` and returned a `RetryingAzureManagementAPIWrapper` then these two methods call into that helper twice.

Reply:
Decide to keep the original one.